.github/workflows/docker-image.yml

```diff
@@ -1453,6 +1453,74 @@ jobs:
         if: always()
         run: sudo podman logs fetchit
 
+  ssh-validate:
+    runs-on: ubuntu-latest
+    needs: [ build, pull-and-archive ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: pull in podman
+        uses: actions/download-artifact@v1
+        with:
+          name: podman-bins
+          path: bin
+
+      - name: replace
+        run: |
+          chmod +x bin/podman
+          sudo mv bin/podman /usr/bin/podman
+
+      - name: Enable the podman socket
+        run: sudo systemctl enable --now podman.socket
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: fetchit-image
+          path: /tmp
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: colors
+          path: /tmp
+
+      - name: Load the image
+        run: sudo podman load -i /tmp/fetchit.tar
+
+      - name: Load the image
+        run: sudo podman load -i /tmp/colors.tar
+
+      - name: tag the image
+        run: sudo podman tag quay.io/fetchit/fetchit-amd:latest quay.io/fetchit/fetchit:latest
+
+      - name: generate ssh assets
+        run: |
+          mkdir -p ~/.fetchit/.ssh
+          cp ./examples/ssh-config.yaml ~/.fetchit/config.yaml
+          echo ${{ secrets.SSH_PRIVATE_KEY }} > ~/.fetchit/.ssh/id_rsa
+          ssh-keyscan github.com > ~/.fetchit/.ssh/known_hosts
+          sudo chmod 0600 -R ~/.fetchit/.ssh 
+
+      - name: generate fetchit config directory
+        run: |
+            mkdir -p /home/runner/.fetchit
+            cp examples/ssh-config.yaml /home/runner/.fetchit/config.yaml
```

Comment:
@djach7 I'm pretty sure I told you to add these lines, but somehow I didn't realize you already had them covered in that step above - sorry about that & please remove lines #L1505 - #L1508 - and I found the mistake!! 
change this line 
`echo ${{ secrets.SSH_PRIVATE_KEY }} > ~/.fetchit/.ssh/id_rsa`
to
`cat ${{ secrets.SSH_PRIVATE_KEY }} > ~/.fetchit/.ssh/id_rsa`

:pray: this should resolve the failure

Reply:
Updated but still failing unfortunately

Reply:
@djach7 I[ notice above](https://github.com/containers/fetchit/pull/259/files#diff-dbade1e5797a7633e024a7685a43a7cbd96ae6fefc6314294807ae34cd83712cL934) the key is created here: `/root/.ssh/id_rsa` so looks like you'll need to add `sudo` to Lines #1501 & #1502 
also, remove lines #1505-1509 - :pray:

Reply:
Updated once again but the github gods have not smiled upon us